### PR TITLE
rename 'static TA' into 'pseudo TA'

### PR DESCRIPTION
**Pull request by etienne-lms**
This renaming prevents confusion of what is a static TA.
Renaming affect core implementation and documentation.

### CODE REVIEW

File: documentation/optee_design.md [hunk 1]
```diff
@@ -524,24 +524,24 @@ TBD
 
 # 12. Trusted Applications
 
-## Static and Dynamic TAs
+## Pseudo TAs and loaded TAs
```
Comment (jenswi-linaro) [inline documentation/optee_design.md]: Perhaps it would be more clear to say *Pseudo TAs  and dynamically loaded TAs*

**Comment by jenswi-linaro** [comment]
With or without my comment addressed.
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by etienne-lms** [comment]
updated. also slightly modified the description.
@jenswi-linaro, please crosscheck this added change.

**Comment by jenswi-linaro** [comment]
Looks good.

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jforissier** [comment]
@etienne-lms there is a typo in your commit description (renaimng). Sorry for not seeing it sooner.

**Pull request merged**